CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * The list of conflicted paths is printed whenever the working copy changes.
   This can be disabled with the `--quiet` option.
 
+* There's a new `mine()` method on the commit type in templates
+
```

Comment:
nit: this seems too informal. Rephrase this a little and elaborate a tad. Maybe:

```
* Commit objects in templates now have a `mine() -> Boolean` method to determine if the email of the commit author matches the current `user.email` setting
```

Reply:
Updated and also added a note that's it's basically the same as the revset function